src/eventDispatcher.js

```diff
@@ -0,0 +1,35 @@
+function EventDispatcher() {
+    this.eventDispatcherMap = {};
+};
+
+EventDispatcher.prototype.bind = function(eventName, handler) {
+    if (this.eventDispatcherMap[eventName]) {
+        this.eventDispatcherMap[eventName].push(handler);
+    } else {
+        this.eventDispatcherMap[eventName] = [handler];
+    }
+};
+
+EventDispatcher.prototype.trigger = function(eventName) {
+    if (this.eventDispatcherMap[eventName]) {
+        for (var i = 0; i < this.eventDispatcherMap[eventName].length; i++) {
+            this.eventDispatcherMap[eventName][i]();
```

Comment:
What would happen in the following scenario:
```
ed.bind("someEvent", function() { throw new Errror("Hello from error"); }
ed.bind("someEvent", function() { console.log("Hello from second function"); }
```
?

You need to handle this case.